src/core/net/dnssd_server.hpp

```diff
@@ -323,6 +375,9 @@ class Server : public InstanceLocator, private NonCopyable
                                          Message &               aMessage,
                                          const Ip6::MessageInfo &aMessageInfo,
                                          Ip6::Udp::Socket &      aSocket);
+    static DnsQueryType     GetQueryTypeAndName(const Header & aHeader,
```

Comment:
seems no change to this method. Why move location? Should we minimize unrelated change?